From 54d27e6f27884aafca8325a93e257bfb0f40660d Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 18 Apr 2012 19:17:12 +0000
Subject: [PATCH 0271/1077] Mantis 0022440: Reduce the number of compiler
 warnings #pragma warning(disable:4290) // A function was declared using
 exception specification. change reverted, use compiler flag /wd4290 instead

git-svn-id: svn://localhost/SelfServiceCommon/trunk@324 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp b/SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp
index 80a166a9..6ecea5e0 100644
--- a/SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp
+++ b/SelfServiceCommon/Interfaces/generic/fsm/mStateMachine.hpp
@@ -43,11 +43,6 @@
  *
  **************************************************************************
 **/
-
-#pragma warning(disable:4290) // A function was declared using exception specification.
-
-
-
 #ifndef MSTATEMACHINE_HPP
 #define MSTATEMACHINE_HPP
 #ifdef SHOW_INCLUDE_FILES
-- 
2.41.0.windows.1

